### ISSUE TITLE
Unable to navigate to a community from a post with VoiceOver

### ISSUE THREAD

**Issue by WestonHanners** [mlemgroup/mlem#92]
**Describe the bug**
Unable to navigate, activating a community header opens the post

**To Reproduce**
Try to navigate to a community via a post.

**Mlem Information:**
0.0.7

**Comment by Sjmarf** [comment]
Closing as being v1 related. This issue may still exist in v2; I haven't tested it. See also #1607.
